src/components/Authentication/Login.js

```diff
@@ -122,7 +128,22 @@ function Login(props) {
               Sign In with Google
             </Button>
           </CardContent>
-          <Divider variant="middle" />
+          <CardContent>
```

Comment:
I believe we can put the facebook button inside the same cardcontent component as the google button?

Reply:
Sounds good, updated in the latest version